src/controllers/update/update.signed.out.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    return res.render(config.UPDATE_SIGNED_OUT_PAGE, {
+      templateName: config.UPDATE_SIGNED_OUT_PAGE,
+      signedOut: true
```

Comment:
Is signedOut the right name here? showBanners seem more like the effect that you are after?

Reply:
would it be more appropriate to have hideBanners, since it will be set to true in this controller?